public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -524,10 +525,10 @@ table
   Of course, our expression might invoke a property or method that has side effects. Angular has no way of knowing that or stopping us.
 
   The expression could call something like `getFoo()`. Only we know what `getFoo()` does.
-  If `getFoo()` changes something and we happen to be binding to that something, we risk an unpleasant experience. Angular may or may not display the changed value. Angular may detect the change and throw a warning error. Our general advice: stick to data properties and to methods that return values and do no more.
+  If `getFoo()` changes something and we happen to be binding to that something, we risk an unpleasant experience. Angular may or may not display the changed value. Angular may detect the change and throw a warning error. Our general advice is to stick to data properties, as well as methods that return values and do no more.
 
   ### Return the proper type
-  The template expression should evaluate to the type of value expected by the target property.
+  The template expression should evaluate the type of value expected by the target property.
```

Comment:
The "to" is important. Maybe this instead:

The template expression should evaluate to a value of the type expected by the target property.

(You could probably come up with better words than mine.)